internal/server/filesystem/server.go

```diff
@@ -76,53 +73,38 @@ func (s *Server) validatePathWindows(pathCtx internal.PathContext, path string)
 	} else if pathCtx == internal.POD {
 		prefix = s.kubeletPodPath
 	} else {
-		return fmt.Errorf("Invalid PathContext: %v", pathCtx)
+		return fmt.Errorf("invalid PathContext: %v", pathCtx)
```

Comment:
maybe add details about the error, like PathContext must be either PLUGIN or POD ?